mindjunkies/forums/models.py

```
@@ -34,76 +39,76 @@ def save(self, *args, **kwargs):
-    reply = models.ForeignKey(
-        ForumReply, on_delete=models.CASCADE, null=True, blank=True
-    )
-    actor = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="forum_actions"
-    )
-    created_at = models.DateTimeField(auto_now_add=True)
-    is_read = models.BooleanField(default=False)
+    def __str__(self):
+        try:
+            return f'{self.author.username} : {self.body[:30]}'
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)